example/example-parallel-bigmap-local-vs-rpc-pack.ts

```diff
@@ -1,7 +1,7 @@
 import { BigMapAbstraction, TezosToolkit, MichelCodecPacker } from '@taquito/taquito';
 import BigNumber from 'bignumber.js';
 
-const rpc = 'https://api.tez.ie/rpc/florencenet';
```

Comment:
The example won't work because KT1PbmJRhHnAR9xdQayCzeBhLve8fVaHPLra is on florencenet